src/adapter/src/coord/sequencer/cluster.rs

```diff
@@ -1002,6 +1002,15 @@ impl Coordinator {
     /// Returns whether the given cluster exclusively maintains items
     /// that were formerly maintained on `computed`.
     pub(crate) fn is_compute_cluster(&self, id: ClusterId) -> bool {
+        if self
+            .catalog()
+            .for_system_session()
+            .system_vars()
+            .enable_unified_clusters()
+        {
+            return true;
+        }
```

Comment:
I find it hard to reason about this, since it's not clear how the callers are using this information. I think it's probably clearer for every _caller_ of this method to check the `enable_unified_clusters` flag? More verbose, but easier to assess whether each callsite is doing the right thing with unification disabled/enabled.

Reply:
Moved to outside of the function.